### DISCUSSION TITLE
My Home BTicino

### DISCUSSION THREAD

**Discussion by GohSte9** [indomus/forum#2364] [GENERALE - Discussioni generiche]
Buongiorno, comincio scusandomi perché non ho capito se in questo forum si parla anche di quella che voi chiamate “domotica buy”. Non sono esperto del settore e sull’onda dell’entusiasmo per l’acquisto di una casa nuova, ho richiesto l’impianto domotico affidandomi al costruttore (primo errore). Mi è stato installato My Home di BTicino e a fronte di un investimento considerevole, sono attualmente molto deluso.

Nella mia ignoranza avevo richiesto il controllo di tapparelle, luci, clima, videocitofono e carichi (pensando fosse riferito a tutte le prese).

Le tapparelle sono l’unica cosa che più o meno funziona, ma non riesco a visualizzarne lo stato, né sul touch (risultano sempre chiuse) né da remoto.
Il clima controlla solo i termostati, e non gli split dell’aria condizionata.
La gestione carichi prevede il distacco di 4 carichi con varie priorità, ma non posso attivarli o disattivarli io, né visualizzarne in consumi.
Il touch non può essere usato come videocitofono in quanto quello del condominio non è BTicino.
Non posso controllare o gestire nessuna presa dell’abitazione e neanche le luci.

Volevo chiedere se qualcuno ha questo impianto, e se c’è modo di integrare in qualche funzioni come il controllo di tutte le prese della casa da remoto, e la visualizzazione dei consumi.

Ma la domanda più importante è: posso far proseguire il lavoro ad un altro professionista, possibilmente specializzato in domotica? Per il momento l’installatore non mi ha rilasciato nessun documento sul lavoro svolto, ne garanzia ne fogli informativi o altro, e l’accesso all’impianto mi è stato fornito da lui tramite un link, quindi suppongo che loro rimangano “amministratori” in qualche modo.

Scusate se sono stato confusionario e grazie in anticipo se qualcuno risponderà.

**Comment by Kyowo87** [comment]


**Reply by GohSte9** [reply]
Ciao, no ma vorrei farlo. È possibile? Non sono molto esperto

**Reply by efestopini** [reply]
non so come inviarti messaggi privati, non vorrei incollare link che magari non posso

**Reply by Kyowo87** [reply]
Tu sei riuscito?

**Reply by indomus** [reply]
sorry, no advertise, anche involontario. sentitevi in privato

**Reply by efestopini** [reply]
Va be pazienza, non vedo come contattare in privato, mi spiace solo che nessuno lo aiuti
E mi sembra esagerato che non si possa neanche scrivere guarda il video di Tizio sulla data piattaforma, in fondo non ho messo un link ad un sito

**Reply by indomus** [reply]
se si affaccia [in chat](https://indomus.it/link/telegram-chat/), poco ma sicuro trova decine di persone che lo usano e lo potrebbero aiutare.
vedi se lo stesso viene fatto proprio sui video del "tizio" che dici tu, prova a scrivere indomus nei commenti, vedi quanto tempo passa prima che ti blocchi. poi ne parliamo.

**Reply by efestopini** [reply]
il "tizio" che dico io sicuramente non darà aiuto gratuitamente, lo fa di lavoro ed è quello che mi sembrava di aver capito che cercasse l'utente, io non voglio creare problemi e chiedo scusa

**Reply by indomus** [reply]
tranquillo, non ne hai creato alcuno :)

**Comment by Kyowo87** [comment]
Ciao sì è possibile sicuramente, anche io dovrò farlo appena ho un attimo di tempo vorrei iniziare.
